jenkins/bootstrap.py

```diff
@@ -345,7 +346,8 @@ def append_result(gsutil, path, build, version, passed):
                 cache = json.loads(gsutil.cat(path, gen))
                 if not isinstance(cache, list):
                     raise ValueError(cache)
-            except ValueError:
+            except ValueError as e:
```

Comment:
this probably needs to be `ve` for pylint to not complain. though I think pylint is being dumb here.